torch/_dynamo/config.py

```diff
@@ -108,6 +108,11 @@
 # Show a warning on every graph break
 print_graph_breaks = False
 
+# This flag will enable logging nodes while
+# the explain backend is traversing the module
+# for debugging-purposes
+explain_print_graphs = False
```

Comment:
Not a general config, should be an arg to the explain backend.

Reply:
Again, this is what I also prefer but that's the the signature of `compiler_fn` that I need to follow:

```
#_dynamo/backends/registry.py
CompilerFn = Callable[[fx.GraphModule, List[torch.Tensor]], CompiledFn]
```